bot/pagination.py

```diff
@@ -218,10 +222,10 @@ async def paginate(
         def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
             """Make sure that this reaction is what we want to operate on."""
             no_restrictions = (
-                # Pagination is not restricted
-                not restrict_to_user
                 # The reaction was by a whitelisted user
-                or user_.id == restrict_to_user.id
+                user_.id == restrict_to_user.id
+                # The reaction was by a moderator
+                or any(role.id in MODERATION_ROLES for role in user_.roles)
```

Comment:
I don't believe the paginator is inherently restricted to guilds. Therefore, this event may get a `User` which doesn't have `roles`.

Reply:
That's a good point. Fixed in 46ff6949 !